front/src/app/error.tsx

```diff
@@ -10,14 +9,5 @@ export default function Error({
     error: Error;
     reset: () => void;
 }) {
-    useEffect(() => {
-        console.error(error);
-    }, [error]);
-
-    return (
-        <Scaffold>
-            <h2>Something went wrong!</h2>
-            <button onClick={() => reset()}>Try again</button>
-        </Scaffold>
-    );
+    return <GenericError error={error} reset={reset} />;
```

Comment:
Any reason for this being it's own component if it just returns GenericError?

Reply:
There were two error.tsx files just now in app/ and app/logs/[id], but I realised it was unnecessary as all nested routes can redirect to the parent error.tsx

forgot to remove this after the refactor, good catch!